README.md

```diff
@@ -128,10 +137,12 @@ docker run -d --name grafana \
       grafana/grafana:7.5.4
 ```
 
-You can now start your NuoDB database with the NuoDB Collector.
-For more info on how to start your database, please refer to the [NuoDB Docker Blog Part I](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i) and the readme for [NuoDB Collector](https://github.com/nuodb/nuodb-collector).
+4. If you haven't already, [start your NuoDB database](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i).
+      
+5. Start the [NuoDB Collector](https://github.com/nuodb/nuodb-collector) daemons for each NuoDB database process.      
+
+   Use the example NuoDB YCSB workload generator given below to explore the various dashboards.
```

Comment:
This looks like a 6th step. Maybe phase as, 

6. "Start the NuoDB YCSB workload generator to explore the various dashboards with live monitoring data."